src/Facility.AspNetCore/Facility.AspNetCore.csproj

```diff
@@ -1,7 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
I don't know if it would happen with this code, but I did have use cases with AspNetCore 3.1 apps where I had a project that was a netstandard library and needed to reference AspNetCore types. One wouldn't be able to do that with these targets.

Reply:
The only package reference to this library in my project is in our net6 app. I'm happy to dig in and try a better way, but dotnet targets and the build system are _mostly_ foreign territory for me these days. It seems tricky.

Reply:
It probably won't be an issue as any new development should be on .NET 6, but if it is, we could revisit when it comes up.